const.go

```diff
@@ -2,6 +2,7 @@ package notary
 
 // application wide constants
 const (
+	IDSize          = 64
```

Comment:
Rather than `IDSize` something more descriptive like `Sha256HexSize`. Makes it obvious how we expect IDs to be generated.